dee-pabst/week5/games-on-rails/app/views/magic8ball/question.html.erb

```diff
@@ -0,0 +1 @@
+<h2><%= @response %></h2>
```

Comment:
you should follow what you had for secret number
a page for the pick and result
same here a question (form) and answer. Something like below

```
<form action="/magic/results" class="magic_fortune">
    <label>
        Ask a yes or no question!
        <input class="magic_button" name="question" placeholder="Is this the real life?" required>
    </label>    

    <button class="magic_button glow-on-hover">Fortune, please!</button>
</form>